src/python/grapl-model-plugin-deployer/src/grapl_model_plugin_deployer.py

```diff
@@ -364,7 +364,7 @@ def upload_plugin(s3_client: BaseClient, key: str, contents: str) -> Optional[Re
 
 
 origin_re = re.compile(
```

Comment:
typecheck seems to not like this

src/grapl_model_plugin_deployer.py:367: error: invalid syntax

Reply:
suspicion: the {2} here is the culprit

Reply:
Makes sense! There appeared to be a couple of issues with the regex string (sadly) due to linter and type checker. I'll probably push another commit up if need be.